pkg/iac-providers/terraform/commons/extract-container-images.go

```diff
@@ -100,15 +100,15 @@ func fetchContainersFromAwsResource(resource jsonObj, hclBody *hclsyntax.Body, r
 			}
 			fileData, err := ioutil.ReadFile(fileLocation)
 			if err != nil {
-				zap.S().Errorf("error fetching containers from aws resource: %v", err)
+				zap.S().Warnf("error fetching containers from aws resource: %v", err)
```

Comment:
if it is just a warning, we probably shouldn't return the control. Also the messages clearly says `error`